en/full-node.md

```diff
@@ -0,0 +1,362 @@
+
+# Running A Full Node
+
+<p class="summary">Learn how run a full node</p>
+
+<div markdown="1" id="toc" class="toc"><div markdown="1">
+
+* Table of contents
+{:toc}
+
+<ul class="reportissue"><li><a href="https://github.com/bitcoin/bitcoin.org/issues/new">Report An Issue</a></li></ul>
+<ul class="editsource"><li><a href="https://github.com/bitcoin/bitcoin.org/tree/master/en/full-node.md">Edit On GitHub</a></li></ul>
+
+</div></div>
+<div markdown="1" class="toccontent">
+
+## What Is A Full Node?
+
+A full node is a program that fully validates transactions and blocks.
+Almost all full nodes also help the network by accepting transactions
+and blocks from other full nodes, validating those transactions and
+blocks, and then relaying them to further full nodes.
+
+Most full nodes also serve lightweight clients by allowing them to
+transmit their transactions to the network and by notifying them when a
+transaction affects their wallet. If not enough nodes perform this
+function, clients won't be able to connect through the peer-to-peer
+network---they'll have to use centralized services instead.
+
+Many people and organizations volunteer to run full nodes using spare
+computing and bandwidth resources---but more volunteers are needed to
+allow Bitcoin to continue to grow.  This document describes how you can
+help and what helping will cost you.
+
+## Costs And Warnings
+
+Running a Bitcoin full node comes with certain costs and can expose you
+to certain risks. This section will explain those costs and risks so you
+can decide whether you're able to help the network.
+
+### Special Cases
+
+Miners, businesses, and privacy-conscious users rely on particular
+behavior from the full nodes they use, so they will often run their own
+full nodes and take special safety precautions. This document does not
+cover those precautions---it only describes running a full node to help
+support the Bitcoin network in general.
+
+Please [consult an expert](/en/community) if you need help setting up
+your full node correctly to handle high-value and privacy-sensitive
+tasks.
+
+### Secure Your Wallet
+
+It's possible and safe to run a full node to support the network and use
+its wallet to store your bitcoins, but you must take the same
+precautions you would when using any Bitcoin wallet.  Please see the
+[securing your wallet page](/en/secure-your-wallet) for more
+information.
+
+### Minimum Requirements
+
+Bitcoin Core full nodes have certain requirements. If you try running a
+node on weak hardware, it may work---but you'll likely spend more time
+dealing with issues. If you can meet the following requirements, you'll
+have an easy-to-use node.
+
+* Desktop or laptop hardware running recent versions of Windows, Mac OS
+  X, or Linux.
+
+* 50 gigabytes of free disk space
+
+* 2 gigabytes of memory (RAM)
+
+* Broadband Internet connection
```

Comment:
Sure, I'll change that.  

For reference, I just checked the stats on my fastest and best connected node (with >99% uptime) and it says the average for the last 30 days is 592 Kbit outgoing and 63 Kbit incoming.  That outgoing is may be a bit high for a low-end DSL connection around here, but the average is skewed by the fact that my connection supports up to 100 Mbit/s and regularly sees peaks of about 14 MBit/s---much higher than supported by most home connections around here.

![bitcoind3](https://cloud.githubusercontent.com/assets/61096/5771071/d369c59e-9d03-11e4-972b-ed0887f7415d.png)

Based on that, I think pretty much any broadband connection would be an asset to the network.  However, a connection with low outgoing speed will more commonly get saturated, giving a poor experience to the volunteer---so I think saying "a good connection" is warranted.